main.go

```diff
@@ -3,24 +3,35 @@ package main
 import (
 	"flag"
 	"fmt"
+	"github.com/huytd/go-play/engine"
+	"github.com/huytd/go-play/web"
 	"io/ioutil"
 	"log"
 	"os"
-
-	"github.com/huytd/go-play/engine"
-	"github.com/huytd/go-play/web"
```

Comment:
In your commit `github.com/huytd/go-play` imports are together with standard library imports, but should be separated :) https://github.com/golang/go/wiki/CodeReviewComments#imports